web/spec/supabase.yml

```diff
@@ -426,6 +426,14 @@ pages:
             data: { hello: 'world' } 
           })
           ```
+        - name: Update a user's email address.
```

Comment:
Hey @yusufcodes - looks like the indentation is a bit off here. Could you please double-check the formatting? Thanks!